test/systemtest/utils.py

```diff
@@ -511,7 +516,8 @@ def filter_remove_sw_test_status_log_prefix(line):
     """
 
     # Example of a full prefix to be matched:
-    # 1629002: (../src/lowrisc_dv_sw_test_status_0/sw_test_status_if.sv:42) [TOP.chip_earlgrey_verilator.u_sw_test_status_if]
+    # 1629002: (../src/lowrisc_dv_sw_test_status_0/sw_test_status_if.sv:42) \
+    # [TOP.chip_earlgrey_verilator.u_sw_test_status_if]
```

Comment:
This line is intentionally long to be exactly what we are trying to match in the pattern below. You can add noqa comments to it if lint complains.

Reply:
Done.